src/node_mime.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SRC_NODE_MIME_H_
+#define SRC_NODE_MIME_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include <string>
+#include <vector>
+#include <utility>
+#include "v8.h"
+#include "env.h"
+#include "base_object-inl.h"
+
+namespace node {
+namespace mime {
+
+enum mime_flags {
```

Comment:
Any reason not to make this a member of MIME (and name it Flags or something)?

Reply:
Can move it but was trying to mimic https://github.com/nodejs/node/blob/master/src/node_url.h#L63 , don't have a preference.

Reply:
Ah, okay.  I guess that one is an outlier, see e.g. env.h for some enums that are class members.